test/integration2/schema/misc.spec.js

```diff
@@ -2191,24 +2210,34 @@ describe('Schema (misc)', () => {
 
           after(() => knex.schema.dropTable(tableName));
 
-          it('should return empty resultset when referencing an existent column', () =>
-            knex(tableName)
+          it('should return empty resultset when referencing an existent column', function () {
+            if (!isSQLite(knex)) {
+              return this.skip();
+            }
+
+            return knex(tableName)
               .select()
               .where(fieldName, 'something')
               .then((rows) => {
                 expect(rows.length).to.equal(0);
-              }));
+              });
+          });
 
-          it('should throw when referencing a non-existent column', () =>
-            knex(tableName)
+          it('should throw when referencing a non-existent column', () => {
```

Comment:
```suggestion
          it('should throw when referencing a non-existent column', function () {
```
We missed that one :smile: